### PR TITLE
Removed unnecessary wording from reliable core info

### PR DESCRIPTION
**Pull request by danieltj27**
I've removed the part about having one theme active at a time because this doesn't feel like it's relevant information here. Whilst it is correct, it doesn't really matter what state plugins and themes are in (active/deactive) as it makes no difference in a backup containing only files.

### CODE REVIEW

**Comment by danieltj27** [comment]
I also update the version control section to make it easier to read.

File: reliability/reliability.md [hunk 1]
```diff
@@ -28,4 +28,6 @@ Uptime monitoring is traditionally done at the server level or by checking one o
...
 
 ## Version Control
-Version control allows a user or host to store, view, or revert changes to individual components of a site over time. It's recommended to provide end users a way to manage their site over version control, as it encourages best practices and is a great extra safety net for their WordPress site.
+Version control is a way of tracking the changes made to files over time, especially for sites running WordPress for example. It allows users to track the revision history of code and to revert or apply changes easily via the command line. It is also a good way to debug your website if something goes wrong as you can quickly restore to a previous version very quickly without doing a full backup.
```
Comment (ataylorme) [inline reliability/reliability.md]: > Version control is a way of tracking the changes made to files over time, especially for sites running WordPress for example.

The end of this doesn't quite sound right to me. Version control is great for all kinds of things, beyond websites and WordPress. Maybe we can mention it is a best practice when working with files, such as code, that change over time.

Comment (ataylorme) [reply reliability/reliability.md]: > if something goes wrong as you can quickly restore to a previous version very quickly without doing a full backup. 

I interpret "doing a full backup" as creating a backup, not restoring one.

Should this be something like "if something goes wrong as you can quickly restore to a previous state of the site code without restoring a full backup. "?

**Review by ataylorme — CHANGES_REQUESTED** [review]
@danieltj27 thank you for the update/suggestions! We've done some work on Reliability since the PR so I resolved the merge conflicts and made a few comments. I think with a few adjustments the update to version control will be a good one.

**Comment by danieltj27** [comment]
@ataylorme no problem. I've pushed my new changes based on your suggestions.

**Review by ataylorme — APPROVED** [review]
Looks good, thank you @danieltj27!

**Comment by ataylorme** [comment]
The version control section has been updated on the [reliability handbook page](https://make.wordpress.org/hosting/handbook/handbook/reliability/)

**Pull request merged**